From b1daa14749341644e48ee153dd4b859eeac990f0 Mon Sep 17 00:00:00 2001
From: Wei-Chiu Chuang <weichiu@cloudera.com>
Date: Mon, 2 May 2016 06:35:42 -0700
Subject: [PATCH 1488/2748] HADOOP-12773. HBase classes fail to load with
 client/job classloader enabled (sjlee)

(cherry picked from commit 58acbf940a92ef8a761208a7a743175ee7b3377d)
(cherry picked from commit 7f8c89c33a216e31c4f7ad04d86db2fa46550b2f)

Change-Id: I2776f8e535a942a41fff669306be5a3c4596db39
---
 ...pache.hadoop.application-classloader.properties |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-common-project/hadoop-common/src/main/resources/org.apache.hadoop.application-classloader.properties b/hadoop-common-project/hadoop-common/src/main/resources/org.apache.hadoop.application-classloader.properties
index 2264920..cbbb887 100644
--- a/hadoop-common-project/hadoop-common/src/main/resources/org.apache.hadoop.application-classloader.properties
+++ b/hadoop-common-project/hadoop-common/src/main/resources/org.apache.hadoop.application-classloader.properties
@@ -50,6 +50,7 @@ system.classes.default=java.,\
   org.xml.sax.,\
   org.apache.commons.logging.,\
   org.apache.log4j.,\
+  -org.apache.hadoop.hbase.,\
   org.apache.hadoop.,\
   core-default.xml,\
   hdfs-default.xml,\
-- 
1.7.9.5

